docs/en/notes/changelog.md

```diff
@@ -1,822 +1,99 @@
 
-#### Bug Fixes
+5. **More documentation and tutorials**. We add a bunch of documentation and tutorials to help users get started more smoothly. Read it [here](https://mmdetection3d.readthedocs.io/en/1.1.x/).
 
-- Fix the incorrect registry name when building RoI extractors (#1460)
-- Fix the potential problems caused by the registry scope update when composing pipelines (#1466) and using CocoDataset (#1536)
-- Fix the missing selection with `order` in the [box3d_nms](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/core/post_processing/box3d_nms.py) introduced by [#1403](https://github.com/open-mmlab/mmdetection3d/pull/1403) (#1479)
-- Update the [PointPillars config](https://github.com/open-mmlab/mmdetection3d/blob/master/configs/pointpillars/hv_pointpillars_secfpn_6x8_160e_kitti-3d-car.py) to make it consistent with the log (#1486)
-- Fix heading anchor in documentation (#1490)
-- Fix the compatibility of mmcv in the dockerfile (#1508)
-- Make overwrite_spconv packaged when building whl (#1516)
-- Fix the requirement of mmcv and mmdet (#1537)
-- Update configs of PartA2 and support its compatibility with spconv 2.0 (#1538)
+### Breaking Changes
 
-#### Contributors
+MMDet3D 1.1.x has gone through big changes to have better design, higher efficiency, more flexibility, and more unified interfaces.
+Besides the changes of API, we briefly list the major breaking changes in this section.
+We will update the [migration guide](../migration.md) to provide complete details and migration instructions.
+Users can also refer to the [compatibility documentation](./compatibility.md) and [API doc](https://mmdetection3d.readthedocs.io/en/1.1.x/) for more details.
 
-A total of 13 developers contributed to this release.
+#### Dependencies
 
-@Xiangxu-0103, @ZCMax, @jshilong, @filaPro, @atinfinity, @Tai-Wang, @wenbo-yu, @yi-chen-isuzu, @ZwwWayne, @wchen61, @VVsssssk, @AlexPasqua, @lianqing11
+- MMDet3D 1.1.x runs on PyTorch>=1.6. We have deprecated the support of PyTorch 1.5 to embrace the mixed precision training and other new features since PyTorch 1.6. Some models can still run on PyTorch 1.5, but the full functionality of MMDet3D 1.1.x is not guaranteed.
+- MMDet3D 1.1.x relies on MMEngine to run. MMEngine is a new foundational library for training deep learning models of OpenMMLab and are widely depended by OpenMMLab 2.0 projects. The dependencies of file IO and training are migrated from MMCV 1.x to MMEngine.
+- MMDet3D 1.1.x relies on MMCV>=2.0.0rc0. Although MMCV no longer maintains the training functionalities since 2.0.0rc0, MMDet3D 1.1.x relies on the data transforms, CUDA operators, and image processing interfaces in MMCV. Note that the package `mmcv` is the version that provides pre-built CUDA operators and `mmcv-lite` does not since MMCV 2.0.0rc0, while `mmcv-full` has been deprecated since 2.0.0rc0.
```

Comment:
We should add one more line:
- MMDet3D 1.1.x is based on MMDet 3.x, which is also a part of OpenMMLab 2.0 projects.